packages/plugin-solidity/index.js

```diff
@@ -0,0 +1,21 @@
+import { SOLIDITY_IMPORT } from '@octolinker/helper-grammar-regex-collection';
+import relativeFile from '@octolinker/resolver-relative-file';
+
+export default {
+  name: 'Solidity',
+
+  resolve(path, [target]) {
+    return relativeFile({ path, target });
+  },
+
+  getPattern() {
+    return {
+      pathRegexes: [/\.sol$/],
+      githubClasses: ['type-solidity', 'highlight-source-solidity'],
```

Comment:
@stefanbuck is the build failing because this is set but no example exists for it in [/packages/blob-reader/fixtures/github.com/issue/code.html](https://github.com/OctoLinker/OctoLinker/blob/2b2c34147fc15099f8059fec0d3953eb8ef0cfa2/packages/blob-reader/fixtures/github.com/issue/code.html)?

Reply:
As far as I remember yes. This HTML is just the output from https://github.com/OctoLinker/testrepo/issues/2. I added a solidity snippet at the end which should make this test pass. The fixture are updated on each CI run https://github.com/OctoLinker/OctoLinker/blob/main/packages/blob-reader/scripts/update-fixtures.sh. Just for debugging purpose we store the output within the repo. I'll take care of updating the fixture.

Reply:
@stefanbuck you missed the ".sol" extension I think

Reply:
@stefanbuck without the `.sol` extension an exception is thrown. Will the code snippet support even do anything with this plugin? I'm trying to test it but I'm not having any luck getting it to link anything for me.

Reply:
Using \`\`\`sol ... results in `highlight-source-gerber`.  \`\`\`solidity ... results in `highlight-source-solidity` so this should work.

Reply:
I'm not to fuzzy about adding those classes. The only purpose is to make linking in code snippets work. @lucaperret feel free to leave the githubClasses array empty

Reply:
I would rather prefer to have it working on code snippet.

Reply:
I need to look into this a bit closer. Unfortunately I don't have capability for this today, but I'll have a look tomorrow evening.